apps/console/lib/stream.ts

```diff
@@ -38,7 +38,6 @@ export const streamServiceMap = async () => {
     const { status } = await call;
     status && console.info("received grpc getAllStream status", status);
   } catch (e) {
```

Comment:
This was the the cause of the empty service bug. Redirects after pipeline saves close and re-open sockets. Socket closes present as exceptions which are handled below this. So we should not be re-initializing services here.